test/e2e/admission_controller/admission_controller_suite_test.go

```diff
@@ -58,6 +60,39 @@ var testPod = v1.Pod{
 	},
 }
 
+var testDeployment = appsv1.Deployment{
+	Spec: appsv1.DeploymentSpec{
+		Replicas: &replicas,
+		Selector: &metav1.LabelSelector{
+			MatchLabels: map[string]string{
+				"app": "sleepjob",
+			},
+		},
+		Template: v1.PodTemplateSpec{
+			ObjectMeta: metav1.ObjectMeta{
+				Labels: map[string]string{
+					"app": "sleepjob",
+				},
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:    "sleepjob",
+						Image:   "alpine:latest",
+						Command: []string{"sleep", "30"},
+					},
+				},
+			},
+		},
```

Comment:
This template will come back in a number of places we might want to factor this out so we can re-use.

Reply:
Done